site/styles/_meta-item-tagline.scss

```diff
@@ -52,6 +52,5 @@
 .meta-detail-content .meta-detail-tagline-and-classification {
   align-items: center;
   display: flex !important;
-  padding-bottom: 12px;
-  padding-top: 5px;
+  padding: 5px 0 16px;
```

Comment:
The GoogleDoc looks like it wants 12px above and below.

Reply:
There was a huddle to figure this one out I think. Theres needed to be some thinking around the desired spacing depending on wheather elements are shown or hidden. What I did decribed in comment[ here ](https://docs.google.com/document/d/1UrEIFVuw24Gm_jjW-5g2ifCVLZE4lpomaGQ2rfQ6Up0/edit#heading=h.ict217zhlxu5)